packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -14,12 +14,13 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { WorkspaceFolder, WorkspaceFoldersChangeEvent, WorkspaceFolderPickOptions, GlobPattern, FileSystemWatcher } from '@theia/plugin';
+import { WorkspaceFolder, WorkspaceFoldersChangeEvent, WorkspaceFolderPickOptions, GlobPattern, FileSystemWatcher, Uri } from '@theia/plugin';
```

Comment:
You do not need `Uri` from `'@theia/plugin'`, you can use `URI` that you imported below

Reply:
fixed